docs/source/migration/3.0.mdx

```diff
@@ -549,9 +549,9 @@ Input.optional(value)
 Optional.presentIfNotNull(value)
 ```
 
-#### Non-optional variables by default
+#### Non-optional variables generation
 
-By default, Apollo Android 3.x represents operation variables that _don't_ have a default value as non-optional Kotlin constructor parameters.
+As with 2.x, Apollo Android 3.x represents operation variables as optional parameters.
```

Comment:
I'd remove that section altogether. If it didn't change, it doesn't belong in the migration guide